cli/src/semgrep/meta.py

```diff
@@ -113,6 +113,11 @@ def repo_name(self) -> str:
         repo_root_str = rev_parse.stdout.strip()
         return str(os.path.basename(repo_root_str))
 
+    @property
+    def repo_display_name(self) -> str:
+        repo_display_name = os.getenv("SEMGREP_REPO_DISPLAY_NAME", self.repo_name)
+        return repo_display_name or self.repo_name
```

Comment:
Redundant to have fallback on both of these lines, but doesn't hurt 😄

Reply:
lol I was resolving pre-commit and didn't think 😂